### ISSUE TITLE
Automated testing is failing during "Install Tripal" phase

### ISSUE THREAD

**Issue by dsenalik** [tripal/tripal#1237]
## BUG/ERROR report
Automated testing is failing for everything during the "Install Tripal" phase with the following error
```
#15 /home/runner/.composer/vendor/drush/drush/drush.php(12): drush_main()
#16 {main}
Error: cution failed: PDOException: SQLSTATE[42P01]: Undefined table:    [error]
7 ERROR:  relation "cv" does not exist
LINE 1: SELECT * FROM cv CV
                      ^ in
/home/runner/work/drupal/includes/database/database.inc:2278
Stack trace:
#0 /home/runner/work/drupal/includes/database/database.inc(2278):
PDOStatement->execute(Array)
```

Reference: pull requests #1233, #1234, #1236

**Comment by laceysanderson** [comment]
Looks like the error you mentioned is a secondary error and the original error is in the chado install phase [here](https://github.com/tripal/tripal/runs/4563106593?check_suite_focus=true#step:7:82).

> 2021-12-17 16:56:30: Calling: tripal_chado_install_chado(Install Chado v1.3)
> Creating 'chado' schema
> [site http://default] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  2252, 0SQLSTATE[42883]: Undefined function: 7 ERROR:  operator does not exist: box @ boxLINE 2: ... 'SELECT * from featureloc where boxquery($1, $2) @ boxrange...                                                             ^HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.:CREATE OR REPLACE FUNCTION featureloc_slice(bigint, bigint) RETURNS setof featureloc AS  'SELECT * from featureloc where boxquery($1, $2) @ boxrange(fmin,fmax)'LANGUAGE 'sql';
> Loading sites/all/modules/tripal/tripal_chado/chado_schema/default_schema-1.3.sql...

**Comment by laceysanderson** [comment]
Wow that took a bit of digging but I finally found the issue:
<img width="661" alt="Screen Shot 2022-01-13 at 1 42 12 PM" src="https://user-images.githubusercontent.com/1566301/149401576-1989827c-6085-44e2-87b0-4880ba1d5c1d.png">

It turns out this function uses an old notation for the "contained in" operator for geometric type (i.e. point, box, lseg, line, path, polygon, and circle) operators that have been deprecated since 8.2. The official PostgreSQL docker container was just updated to Postgresql version 14 which finally removed this deprecated function. All we have to do is change it to `<@` and all supported versions of PostgreSQL will work and our install should be fixed! :-)

**Comment by laceysanderson** [comment]
There. PR submitted :-)

**Comment by spficklin** [comment]
Fixed via PR #1249
